dev/openchpl_preload.sql

```diff
@@ -2278,3 +2278,9 @@ INSERT INTO openchpl.file_type
 (name, description, last_modified_user)
 VALUES
 ('Api Documentation', 'Api Documentation', -1);
+
+INSERT INTO openchpl.filter_type
+	(name, last_modified_user)
+VALUES
+	('Developer Report', -2),
```

Comment:
I think we use -1 as the "inserted by the db script" user id vs our special -2 admin id